### ISSUE TITLE
Fanuc multipath-custom macros

### ISSUE THREAD

**Issue by dsreeves** [mtconnect/adapter#12]
This may not be a particular issue related to the adapter but this seems to be the best place to ask. Anyways I am trying to read the custom macros on a 30i-B but i am only able to read the first path. How can i read another path's (4 in my case) macros without the use of common macros across the cnc? or is this even possible. I have not had much luck finding any info regarding this particular issue. Any help would be greatly appreciated.

**Comment by wsobel** [comment]
I don't believe this version has multi-path support. You would need to set the path each time before you read. First you get the number of paths on the controller and then set the path and read all the variables. Treat each path as if it is another sub-controller–all the data will change for each path on the FANUC.
